features/old/accounts/bulk_operations.feature

```diff
@@ -1,4 +1,4 @@
-@javascript
+@javascript @ajax
```

Comment:
I do not think we need that tag

Reply:
@Martouta

Reply:
Same as https://github.com/3scale/porta/pull/986/files#r304489102 :)

Reply:
Anyway guys feel free to do whatever u consider with that commit, even completely remove it if u consider that's the best idea 👍